### DISCUSSION TITLE
ERROR HTTP 500

### DISCUSSION THREAD

**Discussion by MarthaGil** [chamilo/chamilo-lms#5596] [General]
Hola, mi nombre es Martha me encuentro realizando una investigacion para grado de maestria y estoy trabajando con la plataforma Chamilo en mi propuesta, me encuentro con el error http 500 al momento de crear un foro de presentacion en el curso que estoy creando para mis estudiantes, como puedo solucionar ese error para poder crear los foros, gracias

**Comment by nosolored** [comment]
Hola Martha,
¿Entendemos que estás usando el Campus libre de Chamilo en https://campus.chamilo.org?
Si es así, hay una incidencia ya publicada como tarea para los desarrolladores:
https://github.com/chamilo/chamilo-lms/issues/5550
Parece ser que algún filtro de seguridad bloquea algunos caracteres en el título del foro (quizá alguna tilde o ñ u otro carácter especial)
Inténtalo con otro título que no contenga esos caracteres para ver si te deja crearlo, entretanto se soluciona la incidencia.

Saludos,
